composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "one-and-one/one-and-one-mail-connector",
+  "name": "flagbit/spryker-smart-inbox-connector",
```

Comment:
A change in composer package name will break the spryker twig templating. The module of the namespace should be changed as well for this to be ok.

Reply:
Technically it is possible to use this name for the composer package. Would this be sufficient to guarantee the runtime of the lib?